helix-term/src/commands.rs

```diff
@@ -5982,6 +5964,15 @@ fn jump_to_label(cx: &mut Context, labels: Vec<Range>, behaviour: Movement) {
             doc_mut!(cx.editor, &doc).remove_jump_labels(view);
             return;
         }
+        if let Some(ch) = event.char() {
+            let mut grapheme = Tendril::new();
+            grapheme.push(ch);
+
+            let doc = doc_mut!(cx.editor, &doc);
+            let text = doc.text().slice(..);
+            let overlays = labels_to_overlays(labels.iter().skip(outer).take(alphabet.len()), alphabet, text, Some(grapheme));
```

Comment:
Passing in the char isn't necessary. Instead you can enumerate before passing the iterator to the helper function:

```rust
let overlays = labels_to_overlays(
    labels.iter().enumerate().skip(outer).take(alphabet.len()),
    alphabet,
    doc!(cx.editor, &doc).text().slice(..),
);
doc_mut!(cx.editor, &doc).set_jump_labels(view, overlays);
```